fnConsensus/reactor.go

```diff
@@ -313,6 +313,12 @@ func (f *FnConsensusReactor) getValidatorSet() *types.ValidatorSet {
 func (f *FnConsensusReactor) initRoutine() {
 	var currentState state.State
 
+	if !f.cfg.IsValidator {
```

Comment:
Hmm, this change doesn't really make sense, a non-validator shouldn't be running the vote/commit go-routines. In fact a non-validator shouldn't be running the `initRoutine` either.